README.md

```diff
@@ -50,11 +50,10 @@ Planned features:
     (divide [this x y] (/ x y)))
 
 ;; A proxied implementation of MathService for use via RPC
-(defservice remote-math-service
+(def-remote-service remote-math-service
```

Comment:
`defremoteservice` is, IMO, perhaps weirder but also more conventional and therefore probably less wrong.